bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -393,4 +400,25 @@ public ImageData getComponentData() {
                     .orElse(null))
             .build();
     }
+
+    private void initFeaturedImageBasedProperties() {
+        useFeaturedImage = StringUtils.isEmpty(fileReference) && resource.getChild(DownloadResource.NN_FILE) == null;
```

Comment:
`resource.getChild(DownloadResource.NN_FILE)` is already cached in `fileResource` at L151

Reply:
good catch! Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1593/commits/4b6b12e96eefa240bae5f50be33eb36dc431e515